apptest.py

```
@@ -0,0 +1,82 @@
+    row = cursor.fetchone()
+    conn.close()
+
+    bio = row[0] if row else "No bio"
+
+    # XSS vulnerable output
+    return render_template_string(f"<h1>{username}'s Profile</h1><p>{bio}</p>")
+
+if __name__ == "__main__":
+    # typo and wrong port usage
+    init_db()
+    app.run(debug=Tru, port="5000")
```

Ruff (0.11.9)

Undefined name `Tru`

(F821)